CONTRIBUTING.md

```diff
@@ -84,10 +84,34 @@ Types and functions not in public APIs are not considered part of the guarantee.
 
 #### Backporting
 
-We only accept backports of critical bugs, security issues, or bugs without
-easy workarounds, any
-backport MUST not be breaking for either API or behavioral changes.
-We generally do not accept PRs against older release branches.
+We generally do not accept PRs directly against release branches, while we might
+accept backports of fixes/changes already merged into the main branch.
+
+We generally allow backports of following changes to all supported branches:
```

Comment:
"We generally allow backports of the following kinds of changes to all supported branches:"

Reply:
all _maintained_ branches

Reply:
Thanks done